services/ux-backend/main.go

```diff
@@ -40,6 +41,26 @@ func loadAndValidateServerConfig() (*serverConfig, error) {
 		return nil, fmt.Errorf("Malformed user-defined listening port:  %s. shutting down: %v", listenPortAsString, err)
 	}
 
+	runServerWithTLSAsString := os.Getenv("RUN_SERVER_WITH_TLS")
+	if runServerWithTLSAsString == "" {
+		klog.Infof("UX-Backend-server running in insecure mode")
+		config.runServerWithTLS = false
+	} else {
+		runServerWithTLS, err := strconv.ParseBool(runServerWithTLSAsString)
+
+		if err != nil {
+			return nil, fmt.Errorf("Malformed user-defined run server with TLS:  %s. shutting down: %v", runServerWithTLSAsString, err)
+		}
+		if runServerWithTLS {
+			klog.Infof("UX-Backend-server running in secure mode %t", runServerWithTLS)
+			config.runServerWithTLS = true
+		} else {
+			klog.Infof("UX-Backend-server running in insecure mode")
+			config.runServerWithTLS = false
+		}
```

Comment:
These messages are misleading as they state that we are already running which is wrong. 
```suggestion
		if config.tlsEnabled {
			klog.Info("Server configured to run with TLS")
		} else {
			klog.Info("Server configured to run without TLS")
		}
```